File: calculoIMC.c
#include <stdio.h>

int main() {
    char nome[] = "jose vanderson";
    int idade = 21;
    double peso = 54.0;
    double altura = 164; // em cm
    double imc = peso / (altura * altura);

    printf("%s\n", nome);
    printf("sua idade é %d\n", idade);
    printf("seu peso é %.1f kg\n", peso);
    printf("sua altura é %.0f cm\n", altura);
    printf("seu imc é %f\n", imc);

    return 0;
}

File: CalculoIMC.java
public class CalculoIMC {
    public static void main(String[] args) {
        String nome = "jose vanderson";
        int idade = 21;
        double peso = 54.0;
        double altura = 164; // em cm
        double imc = peso / (altura * altura);

        System.out.println(nome);
        System.out.println("sua idade é " + idade);
        System.out.println("seu peso é " + peso + " kg");
        System.out.println("sua altura é " + altura + " cm");
        System.out.println("seu imc é " + imc);
    }
}
